backend/src/main/java/de/neuefische/teddyscoronadiaries/openweatherapi/model/WeatherTemperature.java

```diff
@@ -0,0 +1,21 @@
+package de.neuefische.teddyscoronadiaries.openweatherapi.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class WeatherTemperature {
```

Comment:
Why not simply `Temperature`?

`Temperatue` post fix doesn't add that much, imho.